src/ValueParsers/MonthNameUnlocalizer.php

```diff
@@ -46,6 +46,10 @@ public function unlocalize( $date ) {
 			if ( !is_string( $search ) ) {
```

Comment:
This check is now redundant, isn't it? The key is either a string or an integer (that cannot be matched with the regex).

Reply:
I guess we could remove it and use `(string)$search` below, but to me that seems less clear (and probably also slightly slower). I’d prefer to keep the separate condition.

Reply:
OK, let's keep it.